sfs/array.py

```diff
@@ -423,16 +460,38 @@ def cube(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
 
 
 def sphere_load(fname, radius, center=[0, 0, 0]):
-    """Spherical secondary source distribution loaded from datafile.
+    """Spherical secondary source distribution loaded from the datafile fname.
 
-    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 position, 1
-    weight) per secondary source located on the unit circle.
+    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 for the cartesian 
+    position vector, 1 for the integration weight) per secondary source located 
+    on the unit circle which is resized by the set radius and shifted to the
+    set center.
 
     Returns
     -------
     `ArrayData`
         Positions, orientations and weights of secondary sources.
 
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        #content of 'test_sphere.txt' is the trivial t-design of
+        #http://neilsloane.com/sphdesigns/dim3/des.3.6.3.txt:
+        #1 0 0 1
+        #-1 0 0 1
+        #0 1 0 1
+        #0 -1 0 1
+        #0 0 1 1
+        #0 0 -1 1
```

Comment:
What about moving those comments out of the plotting code and into the description before (or after) the code?

Then you could use an actual URL and could show the values in nice formatting.

Reply:
good idea, was not thinking about that option when implementing this in the first instance. Changed to the suggested version.